pkg/gui/presentation/commits.go

```diff
@@ -249,6 +256,7 @@ func displayCommit(
 	fullDescription bool,
 	bisectStatus BisectStatus,
 	bisectInfo *git_commands.BisectInfo,
+	youAreHereStr string, // non-empty only if we're rebasing and this is the current commit
```

Comment:
Same here, the string is both for knowing the translated string and a flag whether this particular commit should get the label. A separate bool might be clearer, but is also a bit redundant.

Alternatively, I considered passing in `common.Common` so that we know the translated text, and a bool. However, that would make it more awkward for the test below, which would now have to instantiate a `common.Common`.